src/components/global/Header/HamburgerNavigation.tsx

```diff
@@ -10,38 +13,19 @@ export default function HamburgerNavigation() {
           <li>
```

Comment:
@hubert-deriv nit pick: i suggest mapping < li >s 🙂

Reply:
Reused the NavigationLinks component for this